src/components/home/HomeFooter.jsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import { icTravel, icWish, icSearch, icMassage, icProfile } from 'assets';
+import { Link } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <StyledFooter>
+      <StyledNavToggle>
+        <img src={icSearch} />
+        <span>둘러보기</span>
+      </StyledNavToggle>
+      <Link to={'./Wish'}>
+        <StyledNavToggle>
+          <img src={icWish} />
+          <span>위시리스트</span>
+        </StyledNavToggle>
+      </Link>
+      <Link to={'./'}>
+        <StyledNavToggle>
+          <img src={icTravel} />
+          <span>여행</span>
+        </StyledNavToggle>
+      </Link>
+      <StyledNavToggle>
+        <img src={icMassage} />
+        <span>메세지</span>
+      </StyledNavToggle>
+      <StyledNavToggle>
+        <img src={icProfile} alt="" />
```

Comment:
```suggestion
        <img src={icProfile} />
```
여기 alt="" 빼도 좋을 것 같아!